api/db-cdr/generate-cdr/libproject/devstart.rb

```diff
@@ -113,9 +136,9 @@ def publish_cdr(cmd_name, args)
     # session, or else we would revert the active account after running.
     common.run_inline %W{gcloud auth activate-service-account -q --key-file #{key_file.path}}
 
-    source_dataset = "#{env.fetch(:source_cdr_project)}:#{op.opts.bq_dataset}"
-    ingest_dataset = "#{env.fetch(:ingest_cdr_project)}:#{op.opts.bq_dataset}"
-    dest_dataset = "#{env.fetch(:dest_cdr_project)}:#{op.opts.bq_dataset}"
+    source_dataset = "#{tier[:source_cdr_project]}:#{op.opts.bq_dataset}"
```

Comment:
I don't really understand the implications of .fetch() vs [] so please tell me if these should be fetch.